src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -339,10 +339,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
       {/* @debt Remove back button from Navbar */}
       {hasBackButton && currentVenue?.parentId && parentVenue?.name && (
-        <BackButton
-          onClick={backToParentVenue}
-          locationName={parentVenue.name}
-        />
+        <BackButton onClick={backToParentVenue} />
```

Comment:
Why was `locationName` removed here? We want to keep this on the main platform, it should only be removed on `env/github`.

Reply:
In general, locationname was probably removed because I probably made a mistake in the wacky rebase process. I'll fix all of them now

Reply:
fixed in 088c807569a37b13ce953e79d623431827421b6a